### PR TITLE
ZCS-1784: Fixed validation when adding MTA Trusted Networks (includes ZCS-2873, ZCS-1549)

### PR DESCRIPTION
**Pull request by haitran1696**

### CODE REVIEW

File: WebRoot/js/zimbraAdmin/servers/controller/ZaServerController.js [hunk 1]
```diff
@@ -239,8 +243,9 @@ function (params) {
 						excludeMasks.push(cidrData);
 						
 						for(var j=(numIFs-1);j>=0;j--) {
-							if(ZaIPUtil.isInSubNet(cidrData, locals2[j].ipData)) {
-								throw new AjxException(AjxMessageFormat.format(ZaMsg.ERROR_LOCAL_ADDR_EXCLUDED,[locals2[j].ipData.src, chunks[i]]),AjxException.INVALID_PARAM,"ZaServerController.prototype.validateMyNetworks");
+							//if(ZaIPUtil.isInSubNet(cidrData, locals2[j].ipData)) {
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/servers/controller/ZaServerController.js]: better to remove code if not in use

Comment (haitran1696) [reply WebRoot/js/zimbraAdmin/servers/controller/ZaServerController.js]: Hi @grishick , @silentsakky. I updated the code. Please review and tell me if I need to update anything else

**Comment by haitran1696** [comment]
Hi @grishick , @silentsakky. I updated the code. Please review and tell me if I need to update anything else.

**Comment by silentsakky** [comment]
tabs vs space issues are still not fixed

**Comment by haitran1696** [comment]
Hi all,
I am sorry for asking these questions. This is my first bug fix and I don't know what to process next for an approved pull request.
Do I have to merge this pull request to Github? If yes, which branch should I merge to? Is there anything else that I should do for this bug?

**Comment by fatkudu** [comment]
@haitran1696 we use a modified gitflow process for interacting with GitHUB.  @gordyt this is documented in ssh://git@stash.corp.synacor.com:7999/zimbra/git-util.git as far as I know, are we planning to migrate that to github? (you may be able to look at that on from stash anyway if you have access)

To summarise, the target branch is "develop", however, you should be using aspects of  our modfied gitflow process to ensure that git history is easily comprehensible and that old branches are cleaned up.

**Pull request merged**